includes/context/RequestContext.php

```diff
@@ -201,77 +201,24 @@ public function setUser( User $u ) {
 	 * @return User
 	 */
 	public function getUser() {
-		// Wikia change - begin - @author: Michał Roszka
-		global $wgEnableHeliosExt;
-		$authPath = [];
-
-		if ( $this->user === null && $wgEnableHeliosExt ) {
-			$this->user = \Wikia\Helios\User::newFromToken( $this->getRequest() );
-			$authPath['helios'] = ($this->user !== null ? 'OK' : 'FAIL');
-			if ( $this->user !== null && $this->user instanceof User ) {
-				wfRunHooks( 'UserLoadFromHeliosToken', array( $this->user ) );
-			}
-		}
-		// Wikia change - end
-		// Wikia change - begin - @author: wladek
 		global $wgUserForceAnon;
+		// Wikia change - begin
 		if ( $this->user === null && $wgUserForceAnon ) {
 			$this->user = new User();
-			$authPath['force_anon'] = 'OK';
 		}
 
 		if ( $this->user === null ) {
-		// Wikia change - end
-			$this->user = User::newFromSession( $this->getRequest() );
-			$authPath['media_wiki'] = ($this->user !== null ? 'OK' : 'FAIL');
+			$this->user = User::newFromToken( $this->getRequest() );
+			if ( $this->user->isLoggedIn() ) {
+				wfRunHooks( 'UserLoadFromHeliosToken', [ $this->user ] );
+			}
 		}
-
-		$this->logAuthenticationMethod($this->user, $authPath);
+		// Wikia change - end
 
 		return $this->user;
 	}
 
 	/**
-	 * @param $user User
-	 * @param $authSource array
-	 */
-	private function logAuthenticationMethod($user, $authSource) {
-		if ( count( $authSource ) == 0 || $user === null || !$user->isLoggedIn() ) {
-			return;
-		}
-
-		$sampler = new \Wikia\Util\Statistics\BernoulliTrial( 0.25 );
-
-		// send every 4-th request to InfluxDb
-		if ( $sampler->shouldSample() ) {
```

Comment:
As we discussed on Slack, looks like this logging was only used to track if we were falling back to MW authentication, so leaving removed in favour of logging on the Helios client for now.